bot/utility/utility.py

```diff
@@ -95,6 +96,34 @@ async def about(self, ctx: commands.Context):
         embed.add_field(name="Links:", value=str_links)
         await ctx.send(embed=embed)
 
+    @commands.command(name='howto')
+    @command_log
+    async def howto(self, ctx: commands.Context, topic: Optional[str] = None):
+        """Handler for the `howto` command.
+
+        Explains how to do various things
+
+        Args:
+            ctx (discord.ext.commands.Context): The context in which the command was called.
+            what (Optional[str]): What the user wants to know about
+        """
+        subcommands = {
+            "code": ("How to format code with discord markdown", howto_handler.code),
+        }
+        if topic in subcommands:
+            embed = subcommands[topic][1]()
+        else:
+            if topic:
+                message = "Ich kenne diesen Befehl leider nicht. Hast du vielleicht einen der folgenden gemeint?\n"
+            else:
+                message = "Eine Sammlung nützlicher Befehle:\n"
+            embed = discord.Embed(
+                title = "How to !howto",
+                color = constants.EMBED_COLOR_INFO,
+                description = message + "\n".join([f"`{each}`: {subcommands[each][0]}" for each in subcommands])
+            )
+        await ctx.send(embed=embed)
```

Comment:
I would like it better if the variable `message` wasn't part of the embed and instead just a normal message.
So use `await ctx.send(message, embed=embed)` instead, and make some adaptions to the embed like changing the title of it to something like "Available HowTos:".